lib/applications/schema.ts

```diff
@@ -350,4 +350,14 @@ export default `
   type CreateReplacementApplicationResult {
     ok: Boolean!
   }
+
+  input GenerateApplicantsReportInput {
+    startDate: Date!
+    endDate: Date!
+    columns: [ApplicationsReportColumn!]!
+  }
+
+  type GenerateApplicantsReportResult {
```

Comment:
Can we add a comment here saying that in the future this should also return a link to an S3 file?

Reply:
Done!